heuristics.py

```diff
@@ -0,0 +1,117 @@
+import random
+
+from kubernetes import client, config, watch
+
+from aws_metrics import get_volatility_metrics
+
+SPOT_LABEL = 'spot-instance'
+VOLATILITY_SPAN = 168
+
+
+config.load_kube_config('./config-spark-on-cat')
+v1 = client.CoreV1Api()
+
+def nodes_available():
+    ready_nodes = []
+    for node in v1.list_node().items:
```

Comment:
I did think about this, but justified it since, the number of available nodes is variant in a cluster with volatile nodes, and running a for loop of size N ~= 500 is better than scheduling on an unavailable node